tests/Issues/PhpTagsAddedToBlade/Fixture/php_tags_added_to_blade.blade.php.inc

```diff
@@ -0,0 +1,21 @@
+@extends('layout')
```

Comment:
the extension should can be normal `.php.inc`, I think the demo in https://getrector.org/demo/1ebf90f4-e5c3-644e-996f-dfd195cf29da is not related with extension